### PR TITLE
Fixed pattern to allow mask in card number and expiration date.

### PR DESCRIPTION
**Pull request by ci83rtda**
Pattern never matches as mask was adding and space in card number and a backslash in expiration date

### CODE REVIEW

File: src/CreditCard.vue [hunk 1]
```diff
@@ -22,6 +22,7 @@
                     <label for="name">{{ trans.name.label }}</label>
                     <input maxlength="20"
                         name="name"
+                           id="name"
```
Comment (AfikDeri) [inline src/CreditCard.vue]: Make sure your indentation is correct

File: src/CreditCard.vue [hunk 2]
```diff
@@ -31,8 +32,9 @@
                     <label for="card-number">{{ trans.card.label}}</label>
                     <input type="text"
                         name="card_number"
+                           id="card-number"
```
Comment (AfikDeri) [inline src/CreditCard.vue]: Same issue with the indentation here

File: src/CreditCard.vue [hunk 3]
```diff
@@ -31,8 +32,9 @@
...
                         ref="cardNumber"
-                        pattern="[0-9]*"
+                        pattern="[0-9 ]*"
```
Comment (AfikDeri) [inline src/CreditCard.vue]: why the extra space after the 9?

File: src/CreditCard.vue [hunk 4]
```diff
@@ -45,6 +47,7 @@
                         <label for="expirationdate">{{ trans.expiration.label}} (mm/{{isTwoDigitsYear ? 'yy' : 'yyyy'}})</label>
                         <input type="text"
                             name="expiration_date"
+                               id="expirationdate"
```
Comment (AfikDeri) [inline src/CreditCard.vue]: indentation

File: src/CreditCard.vue [hunk 5]
```diff
@@ -55,8 +58,9 @@
                         <label for="securitycode">{{ trans.security.label}}</label>
                         <input type="text"
                             name="security_code"
+                               id="securitycode"
```
Comment (AfikDeri) [inline src/CreditCard.vue]: indentation

File: src/CreditCard.vue (hunk 3 above)
Comment (ci83rtda) [reply src/CreditCard.vue]: Mask adds a space to card number, pattern doesn't allow it.

File: src/CreditCard.vue (hunk 1 above)
Comment (ci83rtda) [reply src/CreditCard.vue]: Sorry auto-indent from IDE

**Pull request merged**